packages/legend-application-studio/src/stores/LegendStudioRouter.ts

```diff
@@ -42,6 +42,7 @@ export const LEGEND_STUDIO_ROUTE_PATTERN = Object.freeze({
   EDIT_WORKSPACE_ENTITY: `/edit/:${LEGEND_STUDIO_PATH_PARAM_TOKEN.PROJECT_ID}/:${LEGEND_STUDIO_PATH_PARAM_TOKEN.WORKSPACE_ID}/entity/:${LEGEND_STUDIO_PATH_PARAM_TOKEN.ENTITY_PATH}`,
   EDIT_GROUP_WORKSPACE: `/edit/:${LEGEND_STUDIO_PATH_PARAM_TOKEN.PROJECT_ID}/groupWorkspace/:${LEGEND_STUDIO_PATH_PARAM_TOKEN.GROUP_WORKSPACE_ID}/`,
   EDIT_GROUP_WORKSPACE_ENTITY: `/edit/:${LEGEND_STUDIO_PATH_PARAM_TOKEN.PROJECT_ID}/groupWorkspace/:${LEGEND_STUDIO_PATH_PARAM_TOKEN.GROUP_WORKSPACE_ID}/entity/:${LEGEND_STUDIO_PATH_PARAM_TOKEN.ENTITY_PATH}`,
+  PROJECT_DASHBOARD: `/overview/:${LEGEND_STUDIO_PATH_PARAM_TOKEN.PROJECT_ID}?`,
```

Comment:
Please look at the underlying issue https://github.com/finos/legend-studio/issues/1523 implementation plan. I have put some thoughts into the routing and I think that's more appropriate than `/overview`, The new component is virtually a new child application, and it's not just about 1 project, it can be about many projects, so we call it `ProjectDashboard` I believe